src/QMCWaveFunctions/Jastrow/JastrowBuilder.cpp

```diff
@@ -110,6 +107,12 @@ bool JastrowBuilder::addOneBody(xmlNodePtr cur)
   bool success=false;
   ParticleSet* sourcePtcl= (*pa_it).second;
   //use lowercase, to be handled by parser later
+  RadialJastrowBuilder rb(targetPtcl,targetPsi,*sourcePtcl);
+  success = rb.put(cur);
+  return success;
+}
+
+  /*
   tolower(funcOpt);
```

Comment:
Are these commented part all moved to the new builder? If not, please add a comment.